src/App.tsx

```diff
@@ -93,11 +96,11 @@ function App() {
       </div>
       <section className="py-12 main-content md:pt-14 w-[327px] m-auto md:m-w-[90%] md:w-3/4">
         <section className="job-listings flex flex-col gap-10 md:gap-4">
-          {filterListings.length > 0 ? (
-            <JobListing jobsData={filterListings} addFilter={handleAddFilter} />
-          ) : (
-            <JobListing jobsData={jobData} addFilter={handleAddFilter} />
-          )}
+          <JobListing
+            jobsData={activeJobListings}
+            addFilter={handleAddFilter}
+            filters={searchFilters}
+          />
```

Comment:
Cleaned up the code by replacing ternary with  `const activeJobListings = filterListings.length > 0 ? filterListings : jobData;` and then passing the array of activeJobListings